packages/exhibitor-migration-script/README.md

```diff
@@ -0,0 +1,48 @@
+exhibitor-migration-script
+==========================
+
+When a user is updating from DC/OS 1.7 or 1.8 we need to update several values in exhibitor to use the new filesystem
```

Comment:
Just 1.8, since we're only planning to run this script during 1.8 -> 1.9 upgrades.